### ISSUE TITLE
Error Google maps with iOS when title is empty

### ISSUE THREAD

**Issue by SimonTellier** [mattermoran/map_launcher#119]
Hi,
I report a bug.
I have error when i try to launch Google maps with only coordinates and my title string is Empty.
i have something like this on my textfield on google maps : "49.000, 1.200000()"
"()" do a bug on google maps.

**Comment by mattermoran** [comment]
I just made a fix for that. Could you try to install from github? If it works ok I will release to pub.dev
In your pubspec.yaml change to this
```yaml
map_launcher:
  git:
    url: https://github.com/mattermoran/map_launcher
```

**Comment by SimonTellier** [comment]
@mattermoran 
Yes it's work with this version
Good job :)

**Comment by mattermoran** [comment]
Great, published on pub as 2.2.3 :)
